core/src/index.ts

```diff
@@ -22,5 +22,6 @@ import INote = NoteSequence.INote;
 
 export {NoteSequence, INoteSequence, INote};
 
+export * from './data';
```

Comment:
What would you think of:

```
import * as data from './data';
export {data};
```

The `data` package has lots of classes in it, and this puts them under their own namespace. I kind of did this by accident in `midi_io` and `sequences` by putting a bunch of static methods in their own enclosing class. I'm now wondering if it would be better to just have the static methods at the top level and do a similar style of namespace encapsulation as I proposed above.

Reply:
Yes, I like this better.